### ISSUE TITLE
Usman L02E10

### ISSUE THREAD

**Issue by UsmanIjaz** [mikeudacity/robo-pace-betatest#72]
Question 2: 
I do not think that high school or college students  know about the rotational matrix, specially when the target market is global. This is way overboard. 
- An explanation about rotational matrix should be added. What is it's purpose?
- How to calculate a rotational matrix?
- Why it is 2x2 matrix?

**Comment by Shreeyak** [comment]
Yes, rotation matrix' are also not part of the pre-requisites in the course curriculum, so kind of weird having it here. We weren't taught it in university either (electronics and comm.)

**Comment by cangelogamboa** [comment]
Thanks for your comments. I will consider it as I make changes to the course.
